applications/openshift/master/file_groupowner_ip_allocations/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+
+title: 'Verify Group Who Owns The OpenShift SDN Container Network Interface Plugin IP Address Allocations'
+
+description: '{{{ describe_file_group_owner(file="/var/lib/cni/networks/openshift-sdn/.*", group="root") }}}'
+
+rationale: |-
+    CNI (Container Network Interface) files consist of a specification and libraries for
+    writing plugins to configure network interfaces in Linux containers, along with a number
+    of supported plugins. Allowing writeable access to the files could allow an attacker to modify
+    the networking configuration potentially adding a rouge network connection.
```

Comment:
s/rouge/rogue

Reply:
Let's fix this in the CIS benchmark too. IIRC I copy/pasted this directly